cocotb/share/lib/embed/gpi_embed.c

```diff
@@ -228,139 +258,177 @@ int embed_sim_init(gpi_sim_info_t *info)
         }
     }
 
-    PyObject *cocotb_module, *cocotb_init, *cocotb_args, *cocotb_retval;
-    PyObject *simlog_obj, *simlog_func;
-    PyObject *argv_list, *argc, *arg_dict, *arg_value;
+    PyObject *pCocotbModule, *pCocotbInit, *pInitReturn;
+    PyObject *pSimLogObject, *pLogFn, *pFilterFn;
+    PyObject *pArgvList;
```

Comment:
This `pCamelCase` naming convention from my experience is not all that common in cocotb vs what was here before. Are you use `p` to mean `pointer` or `python`? Note that you would never use a `PyObject` by itself, so there's little value in marking it as a pointer.

I'd be inclined to keep the diff small, and leave these names in the same style as they were before

Reply:
That's fine. I was trying to bring it in-line with the naming scheme of other variables, such as pEventFn, pLogHandler, pLogFilter.
It's probably better to do that on a different commit/PR specifically for that.